app/2.0/docs/tools/polymer-json.md

```diff
@@ -190,12 +227,12 @@ And here is a configuration to generate two optimized builds: One bundled and on
   }]
 ```
 
-Here is a configuration to generate a bundled, minified application build with the following bundling options:
+The following configuration generates a bundled, minified application build with bundling options:
```

Comment:
Is "build" (in "application build") a noun or verb? Should this be "application built with..."?